src/qmapshack/gis/trk/CGisItemTrk.cpp

```diff
@@ -630,6 +630,17 @@ QString CGisItemTrk::getInfo(quint32 feature) const
         }
     }
 
+    if(getRating() > 0)
```

Comment:
Can't we do this for all items in IGisItem? Saves us duplicating the code 4 times.

Reply:
Yes, good idea.